addon/components/frost-guage.js

```diff
@@ -0,0 +1,180 @@
+import Ember from 'ember'
+const {isPresent} = Ember
+import computed, {readOnly} from 'ember-computed-decorators'
+import {Component} from 'ember-frost-core'
+import {PropTypes} from 'ember-prop-types'
+import layout from '../templates/components/frost-guage'
+
+function getPercent (max, min, value) {
+  let valInt = parseInt(value, 10)
+  let maxInt = parseInt(max, 10)
+  let minInt = parseInt(min, 10)
+  let val = Math.min(Math.max(minInt, valInt), maxInt)
+  return ((val) / max)
+}
+
+export default Component.extend({
+  layout,
+  attributeBindings: ['style'],
+
+  propTypes: {
+    // options
+    elementPrefix: PropTypes.string,
+    min: PropTypes.number,
+    max: PropTypes.number,
+    direction: PropTypes.oneOf(['clockwise', 'counterclockwise']),
+    orientation: PropTypes.number,
+    data: PropTypes.arrayOf(PropTypes.object).isRequired,
+    size: PropTypes.number,
+    text: PropTypes.string
+  },
+
+  getDefaultProps () {
+    const elementPrefix = 'frost-guage'
+
+    return {
+      elementPrefix: elementPrefix,
+      classNames: [elementPrefix],
```

Comment:
This is not necessary. We are getting this for free with the `ember-frost-core` component